dapper/dapper.proto

```diff
@@ -77,22 +78,22 @@ message Tag {
     repeated Timespan span = 2;
 }
 
+message RelationSpans {
+    // The relations to other keys 
+    repeated RelationSpan spans = 1;
+}
+
 message RelationSpan {
-    // The timespan this value is valid for
-    Relation relation = 1;
+    // The type of the relation
+    string rel_type = 1;
     // The timespan this value is valid for
     Timespan span = 2;
 }
 
-message Relation {
-    // The domain the metadata is associated with
-    string domain = 1;
-    // The key the metadata is associated with
-    string from_key = 2;
-    // The key the metadata is associated with
-    string to_key = 3;
-    // The type of relation
-    string rel_type = 4;
+message SnapshotRelation {
+    string from_key = 1;
+    string to_key = 2;
+    string rel_type = 3;
```

Comment:
Why was domain removed?

Reply:
In this PR, SnapshoRelation becomes a field of KeyMetadataSnapshot which already has domain.

Previously we export Relation alone so we needed domain key.